draft-ietf-rats-corim.md

```diff
@@ -1119,6 +1119,67 @@ An Endorsed Values triple declares additional measurements to add to the ACS.
 {::include cddl/endorsed-triple-record.cddl}
 ~~~
 
+#### Conditional Endorsement Triple {#sec-comid-triple-cond-endors}
+
+The semantics of the Conditional Endorsement Triple is as follows:
```

Comment:
After commuting my change, PR #276, I realised we need to work on Conditional Endorsement Triple.

The subject is a Series of `stateful-environment-record` 

However the predicate should be a series of `measurement-map` NOT a series of Endorsement triples, as the Environment simply does not make sense in the predicate,

Reply:
See revised text.